docs/post-mortems/2019-02-05.md

```diff
@@ -281,7 +281,7 @@ verified: for both the trusted cluster and the build cluster, all the plank-crea
 what is your PodNamespace in the config?
 >
 >Katharine [10:32 AM]  
-`test-pods`: https://github.com/kubernetes/test-infra/blob/master/prow/config.yaml#L41
```

Comment:
This is the same change you made in #15967, and as explained there is not an appropriate change to make.

Reply:
thanks.